platform/cc3200-launchxl/apps/blink.c

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
Yes, i addet this for testing. The LaunchPad CC3200 XL Board has 3 Leds. I would see that the main proc is still running. When the red led stops blinking this indicating me that the firmware was crashed or haltet due an error.